backend/prisma/schema.prisma

```diff
@@ -4,10 +4,18 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-   url      = env("DATABASE_URL")
+  url      = env("DATABASE_URL")
   //url      = "postgresql://postgres:docker@localhost:5432/urefer?schema=public"
 }
 
+enum Referral_Status {
+  OPEN
+  CONTACTED
+  REJECTED
+  HIRED
+  DELETED
+}
```

Comment:
We can add a "CLOSED" enum for when someone else was hired (the JOB POST was closed!) so the rest of the candidates are no longer under consideration

Reply:
you should not be able to add a referral to a closed/deleted job post
and
you should not be able to edit a referral unless it is open